README.md

```diff
@@ -207,15 +198,10 @@ Example:
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+- GitHub: [@Joel-100](https://github.com/Joel-100)
+- Twitter: [@Joel Kugonza](https://twitter.com/JoelKugonza)
+- LinkedIn: [Joel Kugonza](https://www.linkedin.com/in/joel-kugonza/)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
```

Comment:
[OPTIONAL] You did a great job, by implementing the required changes! We appreciate your hard work. So, the previous reviewer forgot to mention that. You need to have some features inside your Future features section. Kindly do not leave them like that: 

 

> 🔭 Future Features
> Describe 1 - 3 features you will add to the project.
> 
>  [new_feature_1]
>  [new_feature_2]
>  [new_feature_3]

You are free to remove this text: Describe 1 - 3 features you will add to the project.

Reply:
Alright thanks!